tern/utils/dockerfile.py

```diff
@@ -27,7 +28,7 @@
 
 # regex strings
 cleaning = '[\t\\\\]'
-bash_var = '[\$\{\}]'
+bash_var = '[\$\{\}]'  # noqa
```

Comment:
Probably replacing both strings with a raw string will not cause an issue. Will file a technical-debt issue for this.

Reply:
Filed issue #224